doc/source/force-constants.rst

```diff
@@ -20,10 +20,10 @@ The :py:attr:`ForceConstants.force_constants <euphonic.force_constants.ForceCons
 
 .. math::
 
-  \phi_{\alpha, {\alpha}'}^{\kappa, {\kappa}'} =
-  \frac{\delta^{2}E}{{\delta}u_{\kappa,\alpha}{\delta}u_{{\kappa}',{\alpha}'}}
+  \Phi_{\alpha {\alpha}'}^{\kappa {\kappa}'}(0,l) =
+  \frac{\partial^{2}E}{{\partial}u_{\kappa\alpha0}{\partial}u_{{\kappa}'{\alpha}'l}}
 
-This describes the change in total energy when atom :math:`\kappa` is displaced in direction :math:`\alpha` and atom :math:`\kappa\prime` is displaced in direction :math:`\alpha\prime`.
+This describes the change in total energy when atom :math:`\kappa` in unit cell :math:`0`` is displaced in direction :math:`\alpha` and atom :math:`\kappa\prime` in unit cell :math:`l` is displaced in direction :math:`\alpha\prime`.
 In Euphonic the force constants are stored in 'compact' form, which means that the minimum required information is stored.
```

Comment:
Not sure this is universally true; the information could be more compact if we used symmetry operations. The following sentence is also a bit long and hard to follow. We can probably explain this better!

Perhaps replacing both sentences with something like:

> This "compact" form of the force constants can be expanded as needed to a "full" 3Nx3N matrix of supercell atom pairs by mapping to an equivalent vector from unit cell 0.

Reply:
Done!